docs/src/main/asciidoc/datasource.adoc

```diff
@@ -3,62 +3,61 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Datasources
+[id="datasources"]
+= Configure datasources in Quarkus
```

Comment:
@MichalMaler & @Sanne - The Vale linter spelling warning on the term `datasources` is creating noise in this PR.

As this is an accepted term in Quarkus, I think we should add an exception to this rule? WDYT?

Reply:
Yeah, I agree :)

Reply:
Well. This is the discussion. We confuse "data source" (noun) with "datasource" (the code artifact.

Technically speaking, the title should be "Configure data sources in Quarkus". Depends on how semantically correct we want to be. Quarkus has habitually used the two forms interchangably, but that doesn't mean it is correct. ;)

(the id can be datasources, or should be datasource if we want to stick with file conventions... ).